src/main/webapp/WEB-INF/views/cars/add.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add car</title>
+</head>
+<body>
+<h1>Provide new car</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/add">
+    Enter manufacturer_id<input type="text" name="manufacturer_id"><br>
```

Comment:
1. what if manufacturer with input id doesn't exist?
2. what if you input "2dasf234dafa" in this field?
3. what if you don't input anything in this field?

Reply:
@AntonPanchishko about @Max-Pochepets comment: 
1) you may handle this in your controller and redirect to some error page in this case(but it is not required in the scope of this task);
2) You may Handle NumberFormatException and show some message near input("invalid input etc. try again)
3) You may use required attribute of input tag;